components/product/ProductCard/components/PriceAndName.tsx

```diff
@@ -0,0 +1,55 @@
+import { formatPrice } from "deco-sites/persono/sdk/format.ts";
+
+interface Props {
+  hasMultiplePrices: boolean;
+  hasDiscount: boolean;
+  productName: string;
+  listPrice: number;
+  price: number;
+  priceCurrency?: string;
+  installments: string | null;
+}
+
+export const PriceAndName = ({
+  hasDiscount,
+  hasMultiplePrices,
+  listPrice,
+  price,
+  productName,
+  installments,
+  priceCurrency = "BRL",
+}: Props) => {
+
+  
+  return (
+    <div class="flex-auto flex flex-col p-4 ">
+      <h2
+        class=" line-clamp-2 text-sm lg:text-lg font-normal  text-black h-10 mb-2"
```

Comment:
Nao precisa ter `text-black` ja é a base do site
```suggestion
        class=" line-clamp-2 text-sm lg:text-lg font-normal h-10 mb-2"
```